src/sentry/api/serializers/snuba.py

```diff
@@ -121,7 +121,7 @@ def value_from_row(row, tagkey):
     return tuple(row[k] for k in tagkey)
 
 
-def zerofill(data, start, end, rollup, allow_partial_buckets=False):
+def zerofill(data, start, end, rollup, allow_partial_buckets=False, fill_default=None):
```

Comment:
Would this be a better default?

```suggestion
def zerofill(data, start, end, rollup, allow_partial_buckets=False, fill_default=[]):
```

Reply:
You can't use mutable objects as default values in python as each call will mutate the default value. https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments has more details.

Reply:
Classic python gotcha

Reply:
JavaScript > Python